src/binding/c/misc_api.txt

```diff
@@ -0,0 +1,200 @@
+# vim: set ft=c:
+
+MPI_Aint_add:
+    .desc: Returns the sum of base and disp
+    .skip: ThreadSafe, Fortran, validate-ANY
+    .seealso: MPI_Aint_diff
+    .impl: direct
+/*
+    Notes:
+    MPI_Aint_Add produces a new MPI_Aint value that is equivalent to the sum of the
+    base and disp arguments, where base represents a base address returned by a call
+    to MPI_GET_ADDRESS and disp represents a signed integer displacement. The resulting
+    address is valid only at the process that generated base, and it must correspond
+    to a location in the same object referenced by base. The addition is performed in
+    a manner that results in the correct MPI_Aint representation of the output address,
+    as if the process that originally produced base had called\:
+    .vb
+        MPI_Get_address((char *) base + disp, &result)
+    .ve
+*/
+{
+    MPI_Aint result;
+    MPIR_FUNC_TERSE_STATE_DECL(MPID_STATE_MPI_AINT_ADD);
+
+    MPIR_ERRTEST_INITIALIZED_ORDIE();
+
+    MPIR_FUNC_TERSE_ENTER(MPID_STATE_MPI_AINT_ADD);
+    result = MPID_Aint_add(base, disp);
+    MPIR_FUNC_TERSE_EXIT(MPID_STATE_MPI_AINT_ADD);
+
+    return result;
+}
+
+MPI_Aint_diff:
+    .desc: Returns the difference between addr1 and addr2
+    .skip: ThreadSafe, Fortran, validate-ANY
+    .seealso: MPI_Aint_add
```

Comment:
It returns `Difference between addr1 and addr2`, not an error code. Can we remove the error section (`.N Error ...`) and mentions that this routine returns the difference?

Reply:
Added `.skip: Errors`